presto-server-rpm/pom.xml

```diff
@@ -0,0 +1,176 @@
+                                    <sources>
+                                        <source>
+                                            <location>
+                                                ${project.build.directory}/${server.tar.package}/plugin
+                                            </location>
+                                        </source>
+                                    </sources>
+                                </mapping>
+                                <mapping>
+                                    <directory>/etc/presto</directory>
+                                    <sources>
+                                        <source>
+                                            <location>src/main/resources/dist/config</location>
+                                        </source>
+                                    </sources>
+                                </mapping>
+                                <mapping>
+                                    <directory>/usr/shared/doc/presto</directory>
+                                    <sources>
+                                        <source>
+                                            <location>${project.build.directory}/${server.tar.package}
+                                            </location>
+                                            <!-- Just include Presto's README for now.-->
+                                            <includes>
+                                                <include>README.txt</include>
+                                            </includes>
+                                        </source>
+                                    </sources>
+                                </mapping>
+                                <!-- Add these mappings so that .spec knows these dirs are to be removed too on rpm -e -->
+                                <mapping>
+                                    <directory>/usr/lib/presto</directory>
+                                </mapping>
+                                <mapping>
+                                    <directory>/usr/lib/presto/lib</directory>
+                                </mapping>
+                            </mappings>
+                            <preinstallScriptlet>
+                                <scriptFile>src/main/resources/scripts/preinstall</scriptFile>
+                            </preinstallScriptlet>
+                            <postinstallScriptlet>
+                                <scriptFile>src/main/resources/scripts/postinstall</scriptFile>
+                            </postinstallScriptlet>
+                            <postremoveScriptlet>
+                                <scriptFile>src/main/resources/scripts/postremove</scriptFile>
+                            </postremoveScriptlet>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+
+    <properties>
```

Comment:
Since this is not a module that contains java code, you should add a property to disable checkstyle:

```
<air.check.skip-checkstyle>true</air.check.skip-checkstyle>
```